### PR TITLE
[CHORE] Fix linter installing ESLint and Prettier

### PR DESCRIPTION
**Pull request by vassalloandrea**
- Remove `prettier-standard`
- Install `eslint` with `standard` rule
- Install `prettier`

### CODE REVIEW

File: containers/products-list/__mocks__/products.js [hunk 1]
```diff
@@ -37,19 +37,19 @@ export const products = [
...
+                    },
+                  ],
+                },
+              },
+            },
+          ],
+        },
+      },
+    },
+  },
```
Comment (rainerdema) [inline containers/products-list/__mocks__/products.js:54]: I'm aware of the usefulness of this formatting rule, but personally, I think that all these commas are stressful 👀 (or maybe it's my OCD? 😂)
It's just an opinion but I think changing the [`Trailing Commas`](https://prettier.io/docs/en/options.html#trailing-commas) to `none` isn't a big problem:
https://github.com/nebulab/solidus_nextjs_frontend/blob/d0d59123a01ab9256774b1a8a569400466e2aea2/.prettierrc#L2

CC @dinohamzic @AlessioRocco for an opinion

**Review by rainerdema — APPROVED** [review]
Great work 💪 for me it's ok I left only a non-blocking comment. Let me know about it 👍

File: containers/products-list/__mocks__/products.js (hunk 1 above)
Comment (dinohamzic) [reply containers/products-list/__mocks__/products.js:54]: Agree, I dislike those commas too. @vassalloandrea what do you think?

Comment (vassalloandrea) [reply containers/products-list/__mocks__/products.js:54]: I usually use them, but is not really important for me 👍  I'll rebase it soon

File: README.md [hunk 2]
```diff
@@ -33,26 +33,27 @@ Use `npm run test:feature` to run E2E tests with the Cypress GUI or simply use `
...
+Solidus NextJS Frontent uses eslint with the standardjs rule and prettier libraries
+as style guide, linter and formatter.
 
-```
-  npm run format
-  npm run lint
-```
+Use `npm run lint` to run the eslint linter
+Use `npm run lint:fix` to run the eslint linter with the autofix option
+Use `npm run lint:format` to run the prettier format on all files.
```
Comment (dinohamzic) [inline README.md:43]: Format -> Formatter ?

File: README.md [hunk 3]
```diff
@@ -33,26 +33,29 @@ Use `npm run test:feature` to run E2E tests with the Cypress GUI or simply use `
 
 ### Running the linter
 
-Solidus NextJS Frontent uses prettier-standard library as style guide, linter and formatter.
-It works as a standlone system and doens't need an `.eslintrc` file.
+Solidus NextJS Frontent uses eslint with the standardjs rule and prettier libraries
+as style guide, linter and formatter.
```
Comment (dinohamzic) [inline README.md:37]: `Frontent` -> `Frontend`

`eslint with the standardjs rule` -> `ESLint with the JavaScript Standard style guide and Prettier as formatter`.

Is the above more correct?

**Review by dinohamzic — APPROVED** [review]

**Pull request merged**